### PR TITLE
localization of template resources #19

### PR DESCRIPTION
**Pull request by n2pro**

### CODE REVIEW

File: assets/js/common-components/lineItems.tpl.liquid [hunk 1]
```diff
@@ -13,7 +13,7 @@
             <a ng-bind="lineItem.name" href="{% raw %}{{ 'product/' + lineItem.productId }}{% endraw %}"></a>
             {% if shop.subscription_enabled %}
             <div ng-if="lineItem.paymentPlan">
-                <span ng-bind="'Subscription with a period: ' + lineItem.paymentPlan.intervalCount + ' ' + lineItem.paymentPlan.interval"></span>
+                <span ng-bind="'{{ 'cart.general.subscript_period' | t }} ' + lineItem.paymentPlan.intervalCount + ' ' + lineItem.paymentPlan.interval"></span>
```
Comment (asvishnyakov) [inline assets/js/common-components/lineItems.tpl.liquid]: Why not subscription_period?

File: locales/en.default.json [hunk 2]
```diff
@@ -555,7 +662,8 @@
             "cart": "Cart"
         },
         "quotes": {
-            "quotes": "Quotes"
+            "quotes": "Quotes",
```
Comment (asvishnyakov) [inline locales/en.default.json]: Nowhere used. Remove

File: snippets/collection-views.liquid [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 <div class="collection-view">
   <a title="Grid view" class="change-view{% unless template contains 'list' %} collection-view--active{% endunless %}" data-view="grid">
-    {{ 'collection-view-grid.svg' | asset_url | img_tag: "Grid View" }}
+    {{ 'collection-view-grid.svg' | asset_url | img_tag: "Grid View1" }}
```
Comment (asvishnyakov) [inline snippets/collection-views.liquid]: What is it? Revert back

File: snippets/footer.liquid [hunk 4]
```diff
@@ -31,71 +31,71 @@
...
-                  <span class="fallback-text">Vimeo</span>
+                  <span class="fallback-text">{{ 'common.social.vimeo' | t }}</span>
                 </a>
               </li>
             {% endif %}
             {% if settings.social_fancy_link != blank %}
               <li>
                 <a class="icon-fallback-text" href="{{ settings.social_fancy_link }}" title="{{ 'layout.footer.store_on_social_network' | t: store_name: shop.name, social_network: 'Fancy' | escape }}">
                   <span class="icon icon-fancy" aria-hidden="true"></span>
-                  <span class="fallback-text">Fancy</span>
```
Comment (asvishnyakov) [inline snippets/footer.liquid]: Brand names is not localizable. Revert back

File: locales/en.default.json [hunk 5]
```diff
@@ -637,7 +755,50 @@
...
+        "error": {
+            "title": "Error",
+            "ok": "Ok"
+        },
+        "guest": {
+            "title": "Sign in to your account",
+            "check_out_faster": "You'll be able to check out faster",
+            "email": "Email address",
+            "password": "Password",
+            "forgot_pass": "Forgot password?",
```
Comment (asvishnyakov) [inline locales/en.default.json]: Use full words always.

File: locales/en.default.json [hunk 6]
```diff
@@ -637,7 +755,50 @@
...
+            "sign_in": "Sign in",
+            "without_sign_in": "Continue without signing in",
+            "after_purchasse": "You'll have the option to register for an account after your purchase.",
```
Comment (asvishnyakov) [inline locales/en.default.json]: register_after_purchase

File: locales/en.default.json [hunk 7]
```diff
@@ -637,7 +755,50 @@
...
+            "register_mow": "register for an account now",
```
Comment (asvishnyakov) [inline locales/en.default.json]: mow → now

File: locales/en.default.json [hunk 8]
```diff
@@ -637,7 +755,50 @@
...
+            "as_guest": "Continue as a guest"
```
Comment (asvishnyakov) [inline locales/en.default.json]: continue_as_guest

File: locales/en.default.json [hunk 9]
```diff
@@ -133,8 +150,7 @@
         "error_title": "Ooops! An error is occured!",
         "error_message": "Please tell us technical details for this error and we will be better!",
         "show_error_details_link_text": "Show me the technical details!",
```
Comment (asvishnyakov) [inline locales/en.default.json]: No exclamation mark in original text

File: locales/en.default.json [hunk 10]
```diff
@@ -215,7 +236,22 @@
             "product_search_empty": "No products found",
             "product_search_progress": "Searching...",
             "categories": "Categories",
-            "products": "Products"
+            "products": "Products",
+            "product_added_request": "Product is added to quote request",
```
Comment (asvishnyakov) [inline locales/en.default.json]: `product_added_request`, `go_request`, `update_request` move to `quotes.general`. `update_request` may replace `quotes.general.update_quote` (it's nowhere used). Move `cart.general.close_cart` and `cart.general.countinue_shop` to `dialog.close` and `dialog.continure_shop`. Replace `products.compare_list.close_dialog` with `dialog.close`

File: locales/en.default.json [hunk 11]
```diff
@@ -215,7 +236,22 @@
...
+            "go_request": "Go to quote request",
+            "update_request": "Updating quote request...",
+            "countinue_shop": "Continue shopping",
+            "product_added": "Product is added to cart",
+            "go_cart": "Go to cart",
+            "update_cart": "Updating cart...",
+            "select_list": "Please select list",
```
Comment (asvishnyakov) [inline locales/en.default.json]: Move `cart.general.select_list` to `wishlist.general.select_list`

File: locales/en.default.json [hunk 12]
```diff
@@ -336,7 +429,8 @@
             "password": "Password",
             "password_confirm": "Confirm Password",
             "submit": "Activate Account",
-            "cancel": "Decline Invitation"
+            "cancel": "Decline Invitation",
+            "or": "or"
```
Comment (asvishnyakov) [inline locales/en.default.json]: → `common.or`

File: locales/en.default.json [hunk 13]
```diff
@@ -360,7 +454,9 @@
             "edit": "Edit",
             "delete": "Delete",
             "delete_confirm": "Are you sure you wish to delete this address?",
-            "no_addresses_message": "You have no addresses yet."
+            "no_addresses_message": "You have no addresses yet.",
+            "or": "or",
```
Comment (asvishnyakov) [inline locales/en.default.json]: → `common.or`

File: locales/en.default.json [hunk 14]
```diff
@@ -637,7 +755,50 @@
...
+            "register_mow": "register for an account now",
+            "as_guest": "Continue as a guest"
+        }
+
+    },
+    "confirmation": {
+        "confirm_email": "Thank you for confirming your email",
+        "click_login_link": "click here to log in",
+        "pass_recovery": "Password recovery",
+        "check_email_to_reset_pass": "Please check your email to reset your password.",
```
Comment (asvishnyakov) [inline locales/en.default.json]: pass → password

File: locales/en.default.json [hunk 15]
```diff
@@ -637,7 +755,50 @@
...
+        "confirm_email_send": "Convirmation Email sent",
```
Comment (asvishnyakov) [inline locales/en.default.json]: sent

File: locales/en.default.json [hunk 16]
```diff
@@ -637,7 +755,50 @@
...
+        "check_email_confirm_his": "Please check your email and confirm your email address.",
```
Comment (asvishnyakov) [inline locales/en.default.json]: its

File: locales/en.default.json [hunk 17]
```diff
@@ -16,7 +16,24 @@
...
+            "google": "Google",
+            "instagram": "Instagram",
+            "tumblr": "Tumblr",
+            "youtube": "YouTube",
+            "vimeo": "Vimeo",
+            "fancy": "Fancy"
+        },
+        "error": {
+            "error": "Error",
+            "error_theme": "Error",
```
Comment (asvishnyakov) [inline locales/en.default.json]: duplicate

File: locales/en.default.json [hunk 18]
```diff
@@ -605,7 +720,9 @@
             "only_left": "Only {{ count }} left!",
             "quantity": "Quantity",
             "add_to_cart": "Add to Cart",
-            "details_tab": "Product details"
+            "details_tab": "Product details",
+            "inc": "Inc VAT",
```
Comment (asvishnyakov) [inline locales/en.default.json]: `inc` → `vat` (it's tax name in most countries, while inc is just short of include)

File: templates/quote-request.liquid [hunk 19]
```diff
@@ -124,7 +124,7 @@
                         <div class="grid-item one-sixth medium-down--one-third text-right" ng-if="displayForStatuses(['New'])">
                             <a class="icon-fallback-text btn-secondary remove" href="" ng-click="removeProductFromQuoteRequest(quoteItem)">
                                 <span aria-hidden="true" class="icon icon-x"></span>
-                                <span class="fallback-text">Remove</span>
+                                <span class="fallback-text">{{ '' | t }}</span>
```
Comment (asvishnyakov) [inline templates/quote-request.liquid]: missed json path

File: templates/quote-request.liquid [hunk 20]
```diff
@@ -165,88 +165,88 @@
...
                         <div class="grid-item" ng-if="customer.addresses.length">
                             <select id="BillingAddressFromCustomerAddresses" name="BillingAddressFromCustomerAddresses" required="required" ng-model="quoteRequest.billingAddress.id" ng-change="selectCustomerAddress('Billing')">
-                                <option disabled="disabled" value="">Select address</option>
+                                <option disabled="disabled" value="">{{ 'quotes.request.billing.select_address' | t }}</option>
                                 <option ng-repeat="address in customer.addresses" ng-value="address.id" ng-selected="address.id === quoteRequest.billingAddress.id" ng-bind="stringifyAddress(address)"></option>
                             </select>
                         </div>
                         <div class="grid-item large--one-half">
-                            <label for="BillingAddressFirstName" style="float: left;">First name</label>
+                            <label for="BillingAddressFirstName" style="float: left;">{{ 'quotes.request.billing.first_name' | t }}</label>
```
Comment (asvishnyakov) [inline templates/quote-request.liquid]: use 'customer.addresses.*'

File: locales/en.default.json [hunk 21]
```diff
@@ -292,8 +281,10 @@
             "company": "Company",
             "request_shipping_quote": "Request a shipping quote",
             "submit_quote": "Submit quote request",
-            "update_quote": "Update quote",
             "remove": "Remove",
+            "product_added_request": "Product is added to quote request",
```
Comment (asvishnyakov) [inline locales/en.default.json]: you copied this, not moved. remove `product_added_request`, `go_request`, `update_request` from `cart.general`

File: locales/en.default.json (hunk 18 above)
Comment (asvishnyakov) [reply locales/en.default.json]: where is updates?

File: locales/en.default.json (hunk 14 above)
Comment (aartiomov) [reply locales/en.default.json]: @asvishnyakov for consistency it should be "check_email_to_reset_password" then

Comment (asvishnyakov) [reply locales/en.default.json]: @aartiomov @n2pro yes, I mean abbreviations should not be used

**Pull request merged**